include/aikido/control/ros/RosJointStateClient.hpp

```diff
@@ -0,0 +1,68 @@
+#ifndef AIKIDO_CONTROL_ROS_ROSJOINTSTATECLIENT_HPP_
+#define AIKIDO_CONTROL_ROS_ROSJOINTSTATECLIENT_HPP_
+#include <string>
+#include <boost/circular_buffer.hpp>
+#include <dart/dynamics/dynamics.hpp>
+#include <ros/ros.h>
+#include <ros/callback_queue.h>
+#include <sensor_msgs/JointState.h>
+
+namespace aikido {
+namespace control {
+namespace ros {
+
+class RosJointStateClient
+{
+public:
+  /// Constructor.
+  /// \param _skeleton Skeleton to read JointState updates for.
+  /// \param _nodeHandle ROS node.
+  /// \param _topicName Name of topic to subscribe to for JointState updates.
+  RosJointStateClient(
+    dart::dynamics::SkeletonPtr _skeleton,
+    ::ros::NodeHandle _nodeHandle,
+    const std::string& _topicName,
+    size_t capacity);
+
+  /// Call all callbacks in mCallbackQueue.
+  void spin();
+
+  /// Returns the last position of each joint.
+  /// \param _metaSkeleton Skeleton to read DOFs from.
```

Comment:
- Docstring doesn't specify "of each joint **in `_metaSkeleton`**.
- Nit: "last" -> "most recent"
- Missing `\return` docstring.